src/github.com/couchbase/sync_gateway/rest/admin_api.go

```diff
@@ -39,12 +43,94 @@ func (h *handler) handleCreateDB() error {
 	return base.HTTPErrorf(http.StatusCreated, "created")
 }
 
+// Take a DB online, first reload the DB config
+func (h *handler) handleDbOnline() error {
+	h.assertAdminOnly()
+	dbState := atomic.LoadUint32(&h.db.State)
+	//If the DB is already trasitioning to: online or is online silently return
+	if (dbState == db.DBOnline || dbState == db.DBStarting) {
+		return nil
+	}
+
+	//If the DB is currently re-syncing return an error asking the user to retry later
+	if (dbState == db.DBResyncing) {
+		return base.HTTPErrorf(http.StatusServiceUnavailable, "Database _resync is in progress, this may take some time, try again later")
+	}
+
+	body, err := h.readBody()
+	if err != nil {
+		return err
+	}
+
+	var input struct {
+		Delay int           `json:"delay"`
+	}
+
+	input.Delay = kDefaultDBOnlineDelay;
```

Comment:
delay values in server config is not currently supported. The DB can be started in offline mode, then brought online with a delay via the REST API.